### ISSUE TITLE
Coveralls doesn't run?

### ISSUE THREAD

**Issue by micheljung** [FAForever/api#72]
https://travis-ci.org/FAForever/api#L1601

```
0.73s$ docker run --link faf-db:db -e FAF_DB_PASSWORD=banana faf-api coveralls
You have to provide either repo_token in .coveralls.yml, or launch via Travis or CircleCI
```

**Comment by micheljung** [comment]
See https://github.com/FAForever/server/blob/develop/.travis.yml#L21

**Comment by micheljung** [comment]
Fixed in 2472f17
